aspnetcore/release-notes/aspnetcore-9/includes/signalr-trimming-aot.md

```diff
@@ -20,7 +20,10 @@ The preceding example produces a native Windows executable of 10 MB and a Linux
 
 #### Limitations
 
-* Only the JSON protocol is currently supported. As shown previously, apps need to use the System.Text.Json source generator. This follows the same approach as Minimal APIs.
-* On a SignalR server, Hub method parameters of type `IAsyncEnumerable<T>` and `ChannelReader<T>` where `T` is a ValueType (i.e. `struct`) aren't supported. These will cause runtime exceptions at startup during development and in the published app. For more information, see [SignalR: Using IAsyncEnumerable&lt;T&gt; and ChannelReader&lt;T&gt; with ValueTypes in native AOT (`dotnet/aspnetcore` #56179)](https://github.com/dotnet/aspnetcore/issues/56179).
-* Client proxy generation is not supported in PublishAot. Client proxy generation will result in warnings at build-time and works with `PublishTrimmed` with no warnings.
-* Custom awaitables on the server are not supported in trimming or AOT. The feature switch `Microsoft.AspNetCore.SignalR.Hub.IsCustomAwaitableSupported` AppContext switch and `SignalRCustomAwaitableSupport` MSBuild property, is disabled by default in trimmed and AOT compiled apps.
+* Only the JSON protocol is currently supported:
+  * As shown in the preceding code, apps that use JSON serialization and Native AOT must use the `System.Text.Json` Source Generator. 
```

Comment:
> As shown in the preceding code

Where is the preceding code?

Reply:
Oh, it's the code link above, nvm 👍

Reply:
[Preview for that section is here](https://review.learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-9.0?view=aspnetcore-9.0&branch=pr-en-us-33341#signalr-supports-trimming-and-native-aot)